npm @backstage/cli create-app fails

Hi! Backstage-cli create-app fails with the following error message.

$ backstage-cli plugin:build
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

lerna ERR! yarn run build stderr:

Error: Cannot find module 'esbuild'
Require stack:
- ~/projects/hello-world/node_modules/rollup-plugin-esbuild/dist/index.js
- ~/projects/hello-world/node_modules/@backstage/cli/dist/cjs/build-022aa68a.js
- ~/projects/hello-world/node_modules/@backstage/cli/dist/cjs/index-8177d57e.js
- ~/projects/hello-world/node_modules/@backstage/cli/dist/index.cjs.js
- ~/projects/hello-world/node_modules/@backstage/cli/bin/backstage-cli

error Command failed with exit code 1.

lerna ERR! yarn run build exited 1 in 'plugin-welcome'
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

--- a/packages/cli/templates/default-app/package.json.hbs
+++ b/packages/cli/templates/default-app/package.json.hbs
@@ -32,7 +32,7 @@
     "prettier": "^1.19.1"
   },
   "resolutions": {
-    "**/esbuild": "0.5.3"
+    "esbuild": "0.6.3"
   },
   "prettier": "@spotify/prettier-config",
   "lint-staged": {

